gdp-src-build/conf/templates/silk.bblayers.conf

```diff
@@ -0,0 +1,7 @@
+include templates/bblayers.inc
+
+# porter specific layer configuration
+BBLAYERS += " \
+  ${TOPDIR}/../meta-renesas \
+  ${TOPDIR}/../meta-renesas/meta-rcar-gen2 \
```

Comment:
To get below error, firstly I changed "genivi-demo-platform" to "genivi-dev-platform.bbappend" as @tom--pollard said. And then start build.

ERROR: Required build target 'genivi-dev-platform' has no buildable providers.
Missing or unbuildable dependency chain was: ['genivi-dev-platform', 'gstreamer1.0-plugins-bad-asfmux', 'faac']